app.rb

```diff
@@ -13,20 +13,48 @@
   get '/greek/:word' do
     content_type :xml
 
-    word = params[:word]
     settings.morpheus.raw(word, **query_options)
   end
 
   get '/latin/:word' do
     content_type :xml
 
-    word = params[:word]
     settings.morpheus.raw(word, latin: true, **query_options)
   end
 end
 
+namespace '/analysis' do
```

Comment:
are you planning on supporting JSON output format? That's what Arethusa currently consumes. If you aren't going to support that, and plan to write a new retriever for Arethusa, then I am dubious if it is even worth the effort for you to maintain the legacy bamboo wrapper here.   Are there any consumers of the Perseids morphology service other than Arethusa?

Reply:
> are you planning on supporting JSON output format?

Yes definitely! I'm writing the XML first. After finishing the XML I'll start working on the JSON.

> Are there any consumers of the Perseids morphology service other than Arethusa?

It's possible that there aren't many (or any) consumers of the XML. But I didn't want to swap out the API without it being backwards compatible and once the code is written I think it should be pretty stable.